core/templates/pages/exploration-editor-page/editor-tab/unresolved-answers-overview/unresolved-answers-overview.component.spec.ts

```diff
@@ -65,7 +64,6 @@ describe('Unresolved Answers Overview Component', function() {
     $uibModal = $injector.get('$uibModal');
     explorationStatesService = $injector.get('ExplorationStatesService');
     improvementsService = $injector.get('ImprovementsService');
-    externalSaveService = $injector.get('ExternalSaveService');
```

Comment:
Can we remove this?

https://github.com/oppia/oppia/blob/b1ba8d11e81b968cddaaa66180f8b787e7c72372/core/templates/pages/exploration-editor-page/editor-tab/unresolved-answers-overview/unresolved-answers-overview.component.spec.ts#L56-L58

Reply:
Done.